plugin/color.py

```diff
@@ -35,10 +34,7 @@ def on_activated_async(self) -> None:
             self.initialize()
 
     def initialize(self, is_retry: bool = False) -> None:
-        if not any(configurations_for_view(self.view)):
-            self.initialized = True  # no server enabled, re-open file to activate feature.
-        sessions = list(sessions_for_view(self.view, 'colorProvider'))
-        if sessions:
+        if self.session('colorProvider'):
```

Comment:
The logic has changed here a bit. Previously `self.initialized = True` was set if no sessions were found. Probably not a big deal. Someone would have to activate file twice really quickly for this to make a difference.